pkg/kubelet/cpumanager/cpu_manager.go

```diff
@@ -174,19 +180,19 @@ func (m *manager) reconcileState() {
 		for _, container := range pod.Spec.Containers {
 			status, ok := m.podStatusProvider.GetPodStatus(pod.UID)
 			if !ok {
-				glog.Warningf("[cpumanager] reconcileState: skipping pod; status not found (pod: %s, container: %s)", pod.Name, container.Name)
+				glog.Errorf("[cpumanager] reconcileState: skipping pod; status not found (pod: %s, container: %s)", pod.Name, container.Name)
```

Comment:
Not sure how to increase log levels in unit tests. I'm also not sure how to increase the log levels in k8s build env. If you have other ideas, lmk. Same comments applies for the changes below on the log-level changes.